R/ResultFlagsDependent.R

```diff
@@ -624,6 +624,11 @@ TADA_AutoFilter <- function(.data) {
 #' @param flaggedonly Boolean argument; the default is flaggedonly = FALSE. When
 #' flaggedonly = TRUE, the function will filter the dataframe to show only the
 #' rows of data flagged as Suspect.
+#' 
+#' @param define Boolean argument; the default is define = TRUE. When define = TRUE,
+#' the function will add an additional column (TADA.MethodQualifierCode) providing 
```

Comment:
Suggest to call the new field:
TADA.MeasureQualifierCode.Def

Reply:
Sounds good to me - I will make that change now.